DICOM/src/Microsoft.Health.Dicom.Anonymizer.CommandLineTool/configuration.json

```diff
@@ -0,0 +1,350 @@
+        {"tag":"(0020,000D)", "method":"refreshUID"}, //Study Instance UID
+        {"tag":"(0008,0030)", "method":"redact"}, //Study Time
+        {"tag":"(0020,0200)", "method":"refreshUID"}, //Synchronization Frame of Reference UID
+        {"tag":"(0018,2042)", "method":"refreshUID"}, //Target UID
+        {"tag":"(0040,A354)", "method":"remove"}, //Telephone Number (Trial)
+        {"tag":"(0040,DB0D)", "method":"refreshUID"}, //Template Extension Creator UID
+        {"tag":"(0040,DB0C)", "method":"refreshUID"}, //Template Extension Organization UID
+        {"tag":"(4000,4000)", "method":"remove"}, //Text Comments
+        {"tag":"(2030,0020)", "method":"remove"}, //Text String
+        {"tag":"(0008,0201)", "method":"remove"}, //Timezone Offset From UTC
+        {"tag":"(0088,0910)", "method":"remove"}, //Topic Author
+        {"tag":"(0088,0912)", "method":"remove"}, //Topic Keywords
+        {"tag":"(0088,0906)", "method":"remove"}, //Topic Subject
+        {"tag":"(0088,0904)", "method":"remove"}, //Topic Title
+        {"tag":"(0062,0021)", "method":"refreshUID"}, //Tracking UID
+        {"tag":"(0008,1195)", "method":"refreshUID"}, //Transaction UID
+        {"tag":"(3008,0250)", "method":"remove"}, //Treatment Date
+        {"tag":"(300A,00B2)", "method":"remove"}, //Treatment Machine Name
+        {"tag":"(3008,0251)", "method":"remove"}, //Treatment Time
+        {"tag":"(0040,A124)", "method":"refreshUID"}, //UID
+        {"tag":"(0040,A352)", "method":"remove"}, //Verbal Source (Trial)
+        {"tag":"(0040,A358)", "method":"remove"}, //Verbal Source Identifier Code Sequence (Trial)
+        {"tag":"(0040,A088)", "method":"redact"}, //Verifying Observer Identification Code Sequence
+        {"tag":"(0040,A075)", "method":"cryptoHash"}, //Verifying Observer Name
+        {"tag":"(0040,A073)", "method":"redact"}, //Verifying Observer Sequence
+        {"tag":"(0040,A027)", "method":"substitute"}, //Verifying Organization
+        {"tag":"(0038,4000)", "method":"remove"}, //Visit Comments
+        {"tag":"DA", "method":"dateshift"},
+        {"tag":"DT", "method":"dateshift"}
+    ],
+
+    "profile": [
+        "MINIMAL_KEEP_LIST_PROFILE"
+    ],
+    "defaultSettings": {
+        "perturb": {
+            "span": "1",
+            "roundTo": 2,
+            "rangeType": "Proportional",
+            "distribution": "Uniform"
+        },
+        "dateShift": {
+            "dateShiftKey": "123",
+            "dateShiftScope": "SeriesInstance",
+            "dateShiftRange": "50"
+        }, // Scope could be SeriesInstance, StudyInstance and SOPInstance    
+        "cryptoHash": {
+            "cryptoHashKey": "123",
+            "cryptoHashFunction": "sha256"
+        }, // function could be sha128, sha256, sha512
+        "redact": {
+            "enablePartialAgesForRedact": false,
+            "enablePartialDatesForRedact": false
+        },
+        "encrypt": {
+            "encryptKey": "123456781234567812345678",
```

Comment:
>123456781234567812345678 [](http://example.com/codeflow?start=27&length=24)

It would be better to add a check before real operation, then remind user to change the encryptKey if it equals to the default key in sample configuration.

Reply:
We can add this check next PR.